Tablet/Mobile-Rounds/App.xaml.cs

```diff
@@ -95,7 +96,25 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
                     // configuring the new page by passing required information as a navigation
                     // parameter
 
+                    //TODO: Remove this...
+                    //var fm = ServiceResolver.Resolve<IFileHandler>();
```

Comment:
This stuff can go away once we have the navigation to get to the admin items page. For now, it is just used to simulate the page. Un-comment to use it...